### PR TITLE
Feature/df 479 map job publication form to publication dto

### PR DESCRIPTION
**Pull request by birom**

### CODE REVIEW

File: alv-portal-ui/src/app/job-publication/job-publication-form/job-publication-form.mapper.ts [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+import { NgbDate } from '@ng-bootstrap/ng-bootstrap';
+
+
+export function mapToJobPublicationFormValue(jobAdvertisement: JobAdvertisement): JobPublicationFormValue {
+  //todo: DF-486 implements this
+  return null;
+}
+
+export function mapToCreateJobAdvertisement(jobPublicationFormValue: JobPublicationFormValue, languageIsoCode: string): CreateJobAdvertisement {
+  const reportToAvam = false;
```
Comment (99999989) [inline alv-portal-ui/src/app/job-publication/job-publication-form/job-publication-form.mapper.ts]: I would rather do it inline, like below.

File: alv-portal-ui/src/app/job-publication/job-publication-form/job-publication-form.mapper.ts [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    countryIsoCode: postAddressFormValue.countryIsoCode,
+    postOfficeBoxNumber: !!postAddressFormValue.postOfficeBoxNumberOrStreet.postOfficeBoxNumber
+      ? postAddressFormValue.postOfficeBoxNumberOrStreet.postOfficeBoxNumber.toString()
+      : null,
+    //todo postOfficeBoxCity, postOfficeBoxPostalCode?
+  };
+}
+
+function formatNgbDate(date: NgbDate) {
+  return date ? `${date.year}-${date.month}${date.day}` : null;
```
Comment (99999989) [inline alv-portal-ui/src/app/job-publication/job-publication-form/job-publication-form.mapper.ts]: Is this really the right format? would get `2019-118`

Comment (birom) [reply alv-portal-ui/src/app/job-publication/job-publication-form/job-publication-form.mapper.ts]: thx, fixed

File: alv-portal-ui/src/app/job-publication/job-publication-form/job-publication-form.mapper.ts (hunk 1 above)
Comment (birom) [reply alv-portal-ui/src/app/job-publication/job-publication-form/job-publication-form.mapper.ts]: fixed

**Pull request merged**